src/components/App/AppSagas.js

```diff
@@ -24,13 +26,14 @@ export function setPageAction(data) {
 export function* loadApp() {
   console.info('Loading app...');
 
-  yield showSpinner('Loading Google Api');
+  // yield showSpinner('Loading Google Api');
   yield googleApiLoader.load();
   yield googleDriveApi.loadDriveApi();
   const rootNote = yield noteStorage.scanDrive();
   yield put(rootNoteFoundAction(rootNote));
-  yield setPageAction(PAGES_ENUM.NOTES);
-  yield hideSpinner();
+  // yield setPageAction(PAGES_ENUM.NOTES);
```

Comment:
Let's remove this commented code.

Reply:
Ok. I leave the comments so it's immediately seen where code changes was made and when needed I can easily uncomment them to restore app working state, so please don't concentrate on deleting commented code, all comments will be deleted once all file/component are refactored. I regularly parsing through files so they will not be left undeleted